packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -15,6 +15,18 @@ export interface GraphQLClient {
    */
   request<T>(query: string | DocumentNode, variables?: { [key: string]: unknown }): Promise<T>;
 }
+
+/**
+ * This type represents errors that can occur in a GraphQL client.
+ * In cases where an error status was sent back from the server (`!response.ok`), the `response` will be populated with details. In cases where a response was never received, the `code` can be populated with the error code (e.g. Node's 'ECONNRESET', 'ETIMEDOUT', etc).
+ */
+export type GraphQLClientError = Partial<ClientError> & {
+  /**
+   * This property is inherited from the NodeJS.ErrnoException type.
+   */
```

Comment:
Maybe just remove. We don't have comments for the other `ClientError` props. Think we're covered with the main type comment?
```suggestion
```